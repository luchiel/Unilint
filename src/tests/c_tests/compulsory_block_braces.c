int main()
{
    if(i == 0)
    {
        return 0;
    }
    else
        return i;
}