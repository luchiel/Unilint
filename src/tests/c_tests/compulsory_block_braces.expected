8:9: block brace expected: { or begin
